custom/icds_reports/management/commands/sql_scripts/fix_past_data_part_4.sql

```diff
@@ -0,0 +1,71 @@
+UPDATE 
```

Comment:
Same comment, I think you can combine file from _4 to _7 into one because the query is mostly same just a few parameters changes

Reply:
group by parameters are different in all of them

Reply:
and also the where clause the aggregation level, and planning to execute this weekend so let's keep it that way